anitya/app.py

```diff
@@ -112,13 +112,17 @@ def after_openid_login(resp):
     login was successful on the OpenID server.
     '''
     default = flask.url_for('index')
+    blacklist = APP.config['BLACKLISTED_USERS']
```

Comment:
Also, in the case of Fedora, you will most likely want to maintain this blacklist in the private repo.
Otherwise we are ending up doing public naming & shaming, which is going to cause a lot of hostility.
Also, will this really stop people? Since it's quite easy to create a new OpenID identity, which will once more allow them to mess it all up.